### DISCUSSION TITLE
Submitting multiple lines at one (cant carry values from one form to the other) MYSQL

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#3529] [Help] [answered]
I am having a problem when I submit multiple lines at once in different forms the data input in the first form will not carry over to the forms below it. Look at my format below. 
![image](https://user-images.githubusercontent.com/80058238/143282941-852f9868-0073-4a2e-b803-496f887045fa.png)

I want to input date and time into form 1 and have that propagate across the other forms so the user does not need to input the same info multiple times. (Date and time are in text format) 

when testing I am using the string to pull in the form 1 date field as shown below.
![image](https://user-images.githubusercontent.com/80058238/143283351-e9268a46-c509-47a7-b927-97ee07065d6b.png)

when testing I can get the date to show up in form 2 as the placeholder but it will not submit to the SQL table it will write a blank.
am i doing something wrong or is this a bug?
![image](https://user-images.githubusercontent.com/80058238/143283777-f8769213-475f-4a2c-9d11-592194e4246f.png)

**Comment by mjashanks** [comment, marked as answer]
So - for inputs whose value that you want to carry across, could you
1. Set the `Field` setting to a non-schema name like `constantDateValue`
2. Make sure all of those inputs are bound to `constantDateValue`
3. On the `Save Row` action, use `constantDateValue` to save to each date value that it needs to

**Reply by KernOil** [reply]
That what it was I was using a Schema name 'date' 
Thank you I appreciate the help!
